GitCommands/Settings/AppSettings.cs

```diff
@@ -1451,6 +1451,12 @@ public static int RecentReposComboMinWidth
             set => SetInt("RecentReposComboMinWidth", value);
         }
 
+        public static bool RecentReposShowBranchName
+        {
+            get => GetBool("RecentReposShowBranchName", false);
```

Comment:
I would set this setting default to `true`. Thoughts?
How expensive is `GitModule.GetSelectedBranchFast`?

Reply:
Setting it to true was my first intention too but then I was not sure about forcing new things on the users